client/src/modules/list/components/Items/model/reducer.js

```diff
@@ -0,0 +1,58 @@
+import { ItemActionTypes } from 'modules/list/components/Items/model/actionTypes';
+
+const items = (state, action) => {
+  switch (action.type) {
+    case ItemActionTypes.ADD_SUCCESS:
+    case ItemActionTypes.CLONE_SUCCESS:
+      return { ...state, items: [action.payload.item, ...state.items] };
+    case ItemActionTypes.TOGGLE_SUCCESS:
+      return {
+        ...state,
+        items: state.items.map(item =>
+          item._id === action.payload.item._id
+            ? {
+                ...action.payload.item,
+                isOrdered: action.payload.item.isOrdered
+              }
+            : item
+        )
+      };
+    case ItemActionTypes.VOTE_SUCCESS:
+      return {
+        ...state,
+        items: state.items.map(item =>
+          item._id === action.payload.item._id
+            ? {
+                ...action.payload.item,
+                voterIds: action.payload.item.voterIds
```

Comment:
we need to check voting reducer cause I can see voterIds here. And we have removed IDS from front layer supposedly 😃

Reply:
You're right - it's unnecessary.  It looks like this is a souvenir from the past...